src/transforms.rs

```diff
@@ -0,0 +1,120 @@
+pub type JsonMap = serde_json::Map<String, serde_json::Value>;
+
+pub fn transform1_to_2(json: JsonMap) -> JsonMap {
```

Comment:
TODO: Integration tests for these.

Reply:
IIRC there is a test(s) in the CLI for this